apps/cms.c

```diff
@@ -822,15 +822,8 @@ int cms_main(int argc, char **argv)
     }
 
     if (operation == SMIME_ENCRYPT) {
-        if (!cipher) {
-#ifndef OPENSSL_NO_DES
```

Comment:
I have one question - Do we need similar build flag changes for AES here?

Reply:
AES is widely supported and does not require a conditional compilation flag like OPENSSL_NO_DES.

While OPENSSL_NO_DES allows OpenSSL to be built without DES support, AES does not have an equivalent conditional flag.

Reply:
still I'd like a feedback of maintainers.

Reply:
Right. It is actually not possible to build without AES, and there is no `OPENSSL_NO_AES` guard.